examples/calendar/backend/privacy/calendar_js/witness_calculator.js

```diff
@@ -0,0 +1,307 @@
+module.exports = async function builder(code, options) {
```

Comment:
are this and file above auto generated or they need review?

Reply:
This and above file are auto generated by running `sh setup.sh` and `sh proof.sh` and not manually written.